### PR TITLE
Add prolinux cluster script to manage e2e test vms

### PR DESCRIPTION
**Pull request by hbinkim**
## Done

- [x] set up e2e test vms on test nodes
- [x] add e2e step on Makefile

## Todo
- verify ceph cluster e2e test on #1 
- image deploy to quay.io
 
### prolinux nodes
- hypersds-node1: k8s master node, using pre-existing k8s installed vm-snapshot (k8s v1.19.8, cri-o, calico)
- hypersds-node2: k8s worker node, using pre-existing k8s installed vm-snapshot (k8s v1.19.8, cri-o, calico)
- hypersds-node3: k8s worker node, using pre-existing k8s installed vm-snapshot (k8s v1.19.8, cri-o, calico)

### ubuntu nodes
- ubuntu-node1: ceph node where ceph cluster will be deployed
- ubuntu-node2: additional ceph node to test OSD addition in the future
- ubuntu-node3: additional ceph node to test OSD addition in the future

### CODE REVIEW

**Comment by hbinkim** [comment]
After updating calico version from `3.14.3` to `3.16.9`, the slow connection problem between the calico manager and k8s api server is disappeared. (Personally I don't think it was the main problem, but) I also changed the pod subnet cidr from `10.244.0.0/16` to `10.233.0.0/16`. 

This PR is ready for review.

File: hack/prolinux_cluster.sh [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+
+    # Check SSH connection
+    for ip in "${CEPH_VMs[@]}"; do
+        if ! is_ssh_success "$ip"; then
+            exit 1
+        fi
+    done
+}
+
+cephClean(){
```
Comment (hyoung-90) [inline hack/prolinux_cluster.sh:238]: It would be nice to have `node` in the function name.

File: hack/prolinux_cluster.sh [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+            exit 1
+        fi
+    else
+        echo "[$VM] [ERROR] '$1' snapshot ($2) is not exist!"
+        exit 1
+    fi
+}
+
+# Check if Kubernetes cluster is alread deployed
+is_deployed_kube(){
```
Comment (hyoung-90) [inline hack/prolinux_cluster.sh:77]: It seems that the function does not check if cluster is deployed.

Comment (hyoung-90) [inline hack/prolinux_cluster.sh:77]: It seems that the function does not check if cluster is deployed.

File: hack/prolinux_cluster.sh (hunk 1 above)
Comment (hyoung-90) [inline hack/prolinux_cluster.sh:238]: It would be nice to have `node` in the function name.

File: hack/cluster.sh [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+
+    # Check SSH connection
+    for ip in "${K8s_VMs[@]}"; do
+        if ! is_ssh_success "$ip"; then
+            exit 1
+        fi
+    done
+}
+
+# Loading requestd cluster if exists, or deploy if not
```
Comment (hyoung-90) [inline hack/cluster.sh:147]: This comment need to be modified.

**Review by hyoung-90 — APPROVED** [review]

**Pull request merged**